openapi/openapi-generator/Dockerfile

```diff
@@ -21,7 +21,7 @@ ARG OPENAPI_GENERATOR_USER_ORG=OpenAPITools
 RUN apt-get update && apt-get -y install git python-pip && pip install urllib3==1.24.2
 
 # Install Autorest
-RUN apt-get update && apt-get -qq -y install libunwind8 libicu57 libssl1.0 liblttng-ust0 libcurl3 libuuid1 libkrb5-3 zlib1g
```

Comment:
I think these are required for autorest/dotnet

Reply:
The only changes are:
- Removed libicu57: could not find equivalent for debian10.
- Replaced  libcurl3 by libcurl4: which is the "offical" replacement. Though there are some mentions of not all software being compatible.

About if the are required, once we have csharp working again we hopefuly be able to put that at rest.

Reply:
Have a look here:
https://github.com/dotnet/core/blob/main/Documentation/linux-prereqs.md

libicu63 is the debian 10 replacement.